Source/WarpX.cpp

```diff
@@ -452,7 +452,8 @@ WarpX::ReadParameters ()
         }
 
         // Check that the coarsening_ratio can divide the blocking factor
-        for (int lev=0; lev<maxLevel(); lev++){
+        const int nlevs_max = maxLevel();
```

Comment:
now that you edit this line, I wonder if it lacks a `+1` (aka the old code lacked a `<=`?).
I'll let someone else cross-check this tomorrow morning :)

Reply:
Actually, makes sense. It's the coarsening to the next level.